Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -177,8 +245,19 @@ public final class ExploreView: UIView {
 
 extension ExploreView: UICollectionViewDelegate {
     public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        let item = sections[indexPath.section].items[indexPath.item]
-        delegate?.exploreView(self, didSelectItem: item, at: indexPath)
+        if indexPath.section < exploreSections.count {
```

Comment:
This only works if we're certain the list of sections provided to this class' method `configure(with:)` is sorted with `exploreSections` first in the list. Are we sure that will always happen?

Reply:
I mean, we can't be sure, but it doesn't really make sense to put them anywhere else because it has such a long feed compared to all the other sections. I feel like the assumption is ok for now, but I agree we could future-proof it a little bit for the next iteration.

Reply:
I've rewritten this with collectionViewDataSource.itemIdentifier(for: indexPath) instead so that it is much clearer what  we are handling.